src/app/componments/header/LocaleButton.types.ts

```diff
@@ -0,0 +1,3 @@
+import { AppLocale } from '../../../context/locale/appLocale.enum';
+
+export type LocaleButtonProps = { localeButtonValue: AppLocale; hasBorder: true | false };
```

Comment:
Dobrze by było, żeby te poszczególne typy propsów występowały w osobnej linii. To ułatwia czytanie kodu.
Możliwe, że będzie trzeba poszukać odpowiedniego konfiga w prettierze/eslint